packages/node_modules/@webex/internal-plugin-ediscovery/test/integration/spec/ediscovery.js

```diff
@@ -1,5 +1,211 @@
+        });
+    });
+
+    describe('Requests with url parameter connect to remote ediscovery and return 4xx response', () => {
+      const validUrl = `https://ediscovery-intb.wbx2.com/ediscovery/api/v1/reports/${reportId}`;
+      const invalidUrl = `https://ediscovery-intz.wbx2.com/ediscovery/api/v1/reports/${reportId}`;
+
+      it('getRemoteContent succeeds with valid url', async () => {
+        await complianceUser.webex.internal.ediscovery.getRemoteContent(validUrl)
+          .then(() => {
+            assert.fail('Expected error response due to invalid report id');
+          })
+          .catch((error) => {
+            assert.equal(error.statusCode, 404);
+            assert.include(error.message, `Report Record with ID[${reportId}] could not be found`);
+          });
+      });
+
+      it('getRemoteContent fails with invalid url', async () => {
+        await complianceUser.webex.internal.ediscovery.getRemoteContent(invalidUrl)
+          .then(() => {
+            assert.fail('Expected error response due to invalid url');
+          })
+          .catch((error) => {
+            assert.equal(error.statusCode, 0);
+          });
+      });
+
+      it('getRemoteContentSummary succeeds with valid url', async () => {
+        await complianceUser.webex.internal.ediscovery.getRemoteContentSummary(validUrl)
+          .then(() => {
+            assert.fail('Expected error response due to invalid report id');
+          })
+          .catch((error) => {
+            assert.equal(error.statusCode, 404);
+            assert.include(error.message, `Report Record with ID[${reportId}] could not be found`);
+          });
+      });
+
+      it('getRemoteContentSummary fails with invalid url', async () => {
+        await complianceUser.webex.internal.ediscovery.getRemoteContentSummary(invalidUrl)
+          .then(() => {
+            assert.fail('Expected error response due to invalid url');
+          })
+          .catch((error) => {
+            assert.equal(error.statusCode, 0);
+          });
+      });
+
+      it('getRemoteContentSummaryBySpaceId succeeds with valid url', async () => {
+        await complianceUser.webex.internal.ediscovery.getRemoteContentSummaryBySpaceId(validUrl, spaceId)
+          .then(() => {
+            assert.fail('Expected error response due to invalid report id');
+          })
+          .catch((error) => {
+            assert.equal(error.statusCode, 404);
+            assert.include(error.message, `Report Record with ID[${reportId}] could not be found`);
+          });
+      });
+
+      it('getRemoteContentSummaryBySpaceId fails with invalid url', async () => {
+        await complianceUser.webex.internal.ediscovery.getRemoteContentSummaryBySpaceId(invalidUrl, spaceId)
+          .then(() => {
+            assert.fail('Expected error response due to invalid url');
+          })
+          .catch((error) => {
+            assert.equal(error.statusCode, 0);
+          });
+      });
+    });
+  });
```

Comment:
Most of these tests check for 404s from ediscovery, with specific messages that show they have hit the service but there was no report/content available. Because of several issue around report creation and expiry, user validation against CI and the asynchronous way in which report content is retrieved from the service, it is difficult to test happy path scenarios without creating unstable tests.